client/templates/tweet-admin-controls.html

```diff
@@ -1,23 +1,23 @@
-<md-button ng-hide="tweet.blocked&&!tweet.display" class="md-icon-button" ng-click="setDeletedStatus({id: tweet.id_str, status: !tweet.deleted})">
+<md-button ng-hide="(tweet.blocked&&!tweet.display)||tweet.deleted" class="md-icon-button" ng-click="setPinnedStatus({id: tweet.id_str, status: !tweet.pinned})" title="Pin tweet">
```

Comment:
yeah but it requires more coding than that, and i thought to put the actions as titles, presumably pin/unpin is understood from whether the star is filled or not